portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,27 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private static List<String> comments;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello World!</h1>");
+    // Add comment messages
+    comments = new ArrayList<>();
+    comments.add("Hi there!");
+    comments.add("How are you?");
+    comments.add("Howdy");
```

Comment:
Sorry for being unclear before. I was trying to point out that, since the comments list never changes, we don't need to re-initialize it on every request. Functions like doGet and init are called for every request, so we should do the initialization outside of them. You can initialize static variables in the same line as the declaration (you might find Arrays.asList [1] useful)

[1] https://docs.oracle.com/javase/8/docs/api/java/util/Arrays.html#asList-T...-